OMCompiler/README.Linux.md

```diff
@@ -10,7 +10,7 @@ sudo apt-get build-dep openmodelica
 git clone --recursive https://openmodelica.org/git-readonly/OpenModelica.git OpenModelica
 cd OpenModelica
 autoconf
-./configure --with-omniORB
```

Comment:
Is this `omniORB` actually the standard way to compile on Linux? I never used it.

Reply:
Yes, you need it if you use the old versions of clients such as OMPython.

Reply:
Okay, so the best way would be `./configure CC=clang CXX=clang++ --with-omniORB`? That of course only works if you have clang.
Or I just leaf it as it was.

Reply:
@sjoelund I reverted the changes on this line. Do you approve the PR?